pkg/database/realm.go

```diff
@@ -369,20 +369,14 @@ func (r *Realm) BeforeSave(tx *gorm.DB) error {
 		if strings.Contains(r.SMSTextTemplate, SMSRegion) {
 			r.AddError("SMSTextTemplate", fmt.Sprintf("cannot contain %q - this is automatically included in %q", SMSRegion, SMSENExpressLink))
 		}
-		if strings.Contains(r.SMSTextTemplate, SMSCode) {
```

Comment:
Do we want to keep these validations, but only if [enslink] is included?

Reply:
removing this is the intent of the PR

Reply:
oh i see we might include both the link and the code as a backup